src/main/java/net/coderbot/iris/gui/element/widget/SliderElementWidget.java

```diff
@@ -103,7 +103,7 @@ public boolean mouseClicked(double mx, double my, int button) {
 
 			return true;
 		}
-		return super.mouseClicked(mx, my, button);
+		return false;
```

Comment:
Suggestion: Add a comment here saying why this shouldn't defer to the parent - just for anyone reading the code in the future.